typegate/src/engine/typecheck/common.ts

```diff
@@ -45,6 +45,16 @@ const formatValidators: Record<StringFormat, FormatValidator> = {
   phone: validator.isMobilePhone, // ??
   date: validator.isDate,
   "date-time": validator.isISO8601,
+  name: (value: string) => validator.isAlpha(value.replace(/\s+/g, "")),
+  password: (_: string) => true,
+  gender: (value: string) =>
+    ["male", "female", "other"].includes(value.toLowerCase()),
+  firstname: validator.isAlpha,
```

Comment:
issue: some folks might have numbers in their names or non-latin utf-8 chars. This goes for a lot of these here.

Reply:
yeah, that makes sense, I will ensure that if the StringFormat won't get removed...

Reply:
Done.